pyproject.toml

```diff
@@ -53,9 +53,6 @@ dependencies = [
     # nonblocking file IO
     "aiofiles",
 
-    # Better than dataclasses for NodeIdentifier
```

Comment:
lmao

Reply:
It _probably_ was faster in 3.10 or 3.9 when I did this

Reply:
oh I'm sure, just very funny